cmd/metal-api/internal/service/machine-service.go

```diff
@@ -1525,7 +1525,7 @@ func freeMachine(ds *datastore.RethinkStore, publisher bus.Publisher, ipamer ipa
 		return err
 	}
 
-	deleteEvent := metal.MachineEvent{Type: metal.DELETE, Old: m}
+	deleteEvent := metal.MachineEvent{Type: metal.DELETE, OldMachineID: m.ID}
```

Comment:
is this the only occurrence of MachineEvent, if yes, great

Reply:
Yes :)